index.md

```diff
@@ -0,0 +1,64 @@
+- Be able to write simple Python program
+- Be familiar with Python’s documentation
+- Be familiar with the basic scientific & data science Python modules and know which is most appropriate in each situation
+
+## About the Instructor
+![Leopold](/assets/img/leopold.jpg){:style="max-width:30%;"}
+
+Leopold graduated from ICME at Stanford with a Ph.D. in Computational and Mathematical Engineering in 2021. During his Ph.D., Léopold worked with Prof. Darve on developing new approximate direct solvers for very large systems of equations. Since 2021, Leopold works as a Software Engineer at NVIDIA, where he primarily works on the cuFFT and cuFFTMp libraries. He focuses on developing fast distributed FFT algorithms for efficiently using thousands of GPUs simultaneously. He also worked on DL frameworks such as JAX.
+
+## About the Workshop Assistant
+![Tianyu](/assets/img/tianyu.jpg){:style="max-width:30%;"}
+
+Tianyu is a Ph.D. student in the Institute for Computational and Mathematical Engineering (ICME) at Stanford University working with Prof. Susan Athey and Prof. Daniel McFarland. After receiving his Bachelor's degree in economics and mathematics at University of Toronto, Tianyu received his master's degree in Management Science and Engineering (MS&E) at Stanford. His research has focused on the application of machine learning methods on social sciences such as career trajectories of workers.
+
+# Workshop Materials
+
+## Audience
+This class assumes no prior knowledge of Python. However, familiarity with fundamental programming concepts is required before attending a Python workshop. Participants should have a grasp of concepts such as variables, data types, loops, conditional statements, and functions. Prior exposure to any programming language will help participants quickly adapt to Python syntax and programming principles.
+
+For instance, a student that took a statistics class using R for homeworks is ready for this class. On the other hand, a student with no programming experience whatsoever will find this class very difficult.
+
+## Requirements
+To attend this class, you’ll need a laptop or a desktop computer (Windows, Mac or Linux) with Zoom and a recent web browser. No installation of Python will be required. More detailed instructions will be provided prior to the class.
+
+## Schedule
+This class will focus on the basic building blocks for Python in scientific computing and data science. Specifically, we will cover:
+
+#### Session 1 (July 24, Monday, 1:00 P.M. - 4:00 P.M. PST)
+- General Python
+  - What is Python? When is it useful?
+  - Variables
+  - Conditionals, loops
+  - Containers
+  - Functions
+  - How to read Python’s documentation
+
+#### Session 2 (July 25, Tuesday, 1:00 P.M. - 4:00 P.M. PST)
+- Numpy
+  - What is Numpy and why do we need it?
+  - Arrays
+  - Reshaping, slicing, broadcasting
+- Pandas
+  - Dataframes
+  - Pivots
+- The Python plotting ecosystem
+  - Topic 1 with example time-allocation.
```

Comment:
Are we missing something here?

Reply:
I have added the visualization topic.

Reply:
I basically copied-pasted things from the syllabus, feel free to add new contents to `index.md` after merging.

Reply:
Sg.